### PR TITLE
Use @babel/core

### PR DESCRIPTION
**Pull request by MattiasBuelens**
Change peer dependency to `@babel/core` and update examples. Similar to babel/gulp-babel#133 and babel/babel-loader#522.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,12 +28,12 @@
...
   "license": "ISC",
   "peerDependencies": {
-    "babel-core": "6 || 7 || ^7.0.0-alpha || ^7.0.0-beta || ^7.0.0-rc"
+    "@babel/core": "7 || ^7.0.0-beta || ^7.0.0-rc"
   },
   "devDependencies": {
-    "babel-core": "7.0.0-beta.0",
-    "babel-polyfill": "7.0.0-beta.0",
-    "babel-preset-env": "2.0.0-beta.0",
+    "@babel/core": "7.0.0-beta.5",
```
Comment (Bloomca) [inline package.json]: It is already at 36 :)

**Comment by Bloomca** [comment]
Hey, can we somehow merge it? As I understand, right now it is impossible to use setup for babel@7, since we use old babel-core, which is incompatible with new API (at least my tests fail, and it is my conclusion from debugging).

File: package.json (hunk 1 above)
Comment (MattiasBuelens) [reply package.json]: And in the meantime beta 37 was released. Gotta go fast! 😄

**Comment by mbark** [comment]
Also wondering what the status for this is: it looks to me like the PR is done -- or this there anything left to be done to get it merged?

**Comment by shuhei** [comment]
Sorry for keeping you waiting. I'll publish a beta version soon.

**Comment by shuhei** [comment]
Published `8.0.0-beta.0`. Thank you for your contribution!

**Pull request merged**